src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+// const buttons = [
+//   'AC','+/-','%','÷','7','8','9','x','4','5','6','-','1','2','3','+','0','.','=',
+// ]
+
+// function Display() {
+//     return (
+//        buttons.map(item => <button>{item}</button>)
+//     )
+// }
+
```

Comment:
- Hey, let's remove these commented pieces of code to make our codebase clean and maintainable.

Reply:
- Hey, let's also remove all the boilerplate that got created by the CRA (texts, images, and styles).
   - index.css
   - logo.svg
They should all be deleted.